### PR TITLE
ARP Part 2: Update Switch Properties

### PR DESCRIPTION
**Pull request by niksv**
Part of #3118

### CODE REVIEW

File: services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java [hunk 1]
```diff
@@ -287,28 +287,61 @@ public SwitchPropertiesDto updateSwitchProperties(SwitchId switchId, SwitchPrope
...
+
+    private void validateSwitchProperties(SwitchId switchId, SwitchProperties updatedSwitchProperties) {
+        if (!updatedSwitchProperties.isMultiTable()) {
+            if (updatedSwitchProperties.isSwitchLldp()) {
+                throw new IllegalSwitchPropertiesException(
+                        String.format("Illegal switch properties combination for switch %s. 'switchLldp' property "
+                                + "can be set to 'true' only if 'multiTable' property is 'true'.", switchId));
+            }
+
+            if (updatedSwitchProperties.isSwitchArp()) {
```
Comment (dpoltavets) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: Can we combine this condition and the condition above?

Comment (niksv) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: fixed

File: src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java [hunk 2]
```diff
@@ -288,28 +288,59 @@ public SwitchPropertiesDto updateSwitchProperties(SwitchId switchId, SwitchPrope
...
+                    + "can be set to 'true' only if 'multiTable' property is 'true'.";
+            if (updatedSwitchProperties.isSwitchLldp()) {
+                throw new IllegalSwitchPropertiesException(String.format(propertyErrorMessage, switchId, "switchLldp"));
+            }
+
+            if (updatedSwitchProperties.isSwitchArp()) {
+                throw new IllegalSwitchPropertiesException(String.format(propertyErrorMessage, switchId, "switchArp"));
+            }
+
+            List<String> flowsWitchEnabledArp = flowRepository.findByEndpointSwitchWithEnabledArp(switchId).stream()
```
Comment (rozdy) [inline src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: witch -> which or with

Comment (niksv) [reply src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/SwitchOperationsService.java]: fixed

**Pull request merged**